README.es.md

```diff
@@ -0,0 +1,135 @@
+
+Porfavor tenga en cuenta que no es necesario para ejercer plenamente la definición en cada archivo de prueba - Es suficiente con probar solo los elementos globalmente accesibles en la prueba de archivos globar y ejercer la definición en el módulo del archivo de prueba, o viceversa.
+
+#### ¿Qué pasa con paquetes scoped?
+
+Types para un paquete scoped `@foo/bar` deberán ir en `types/foo__bar`. tenga en cuenta el doble guión bajo.
+
+Cuando `dts-gen` es utilizado como scaffold en un paquete scoped, las propiedades `paths` deberan ser adaptadas manualmente en el paquete generado
+`tsconfig.json` para referenciar correctamente el paquete scoped:
+
+```json
+{
+    "paths":{
+      "@foo/bar": ["foo__bar"]
+    }
+}
+```
+
+
+#### El historial de archivos en GitHub parece incompleto.
+
+GitHub no le hace [support](http://stackoverflow.com/questions/5646174/how-to-make-github-follow-directory-history-after-renames) historial de archivos para archivos renombrados. Utilice [`git log --follow`](https://www.git-scm.com/docs/git-log) en su lugar.
+
+#### Debería añadir un espacio de nombres vacío que no exporte un módulo que utilice ES6 style imports?
+
+Algunos paquetes, como [chai-http](https://github.com/chaijs/chai-http), exportan una función.
+
+importar este módulo con un ES6 style import de forma `import * como foo de "foo";` conduce al error:
+
+> error TS2497: El módulo 'foo' resuelve a una entidad non-module y no puede ser importado utilizando esta construcción
+
+Este error puede ser suprimido merging la declaración de función con un espacios de nombres vacíos de el mismo nombre, pero esta práctica no es recomendable.
+Esto es un citado común [Respuestas de Stack Overflow](https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this) con respecto a este asunto.
+
+Es más apropiado importar este módulo utilizando la sintaxis `import foo = require("foo");`, o utilizando una importación predeterminada como `import foo from "foo";` if using the `--allowSyntheticDefaultImports` flag if your module runtime supports an interop scheme for non-ECMAScript modules as such.
+
+## Licencia
+
+Este proyecto es licenciado bajo la licencia MIT.
+
+Los derechos de autor definidos en los archivos son respectivos de cada contribuidor enumerado al comienzo de cada archivo de definición.
```

Comment:
Change to:
Los derechos de autor de cada archivo de definición son respectivos de cada contribuidor listado al comienzo de cada archivo de definición.